ci/quality-base.sh

```diff
@@ -33,7 +33,10 @@ run_pyproject_fmt() {
 }
 
 run_bandit() {
-    run_uvx bandit --configfile pyproject.toml --quiet --recursive $(python_files_and_folders)
+    # Bandit doesn't support Python 3.14 yet.
```

Comment:
Do we have a ticket on our own backlog to bump Bandit when upstream is fixed?

Reply:
We do now :-) https://github.com/ICTU/quality-time/issues/12125